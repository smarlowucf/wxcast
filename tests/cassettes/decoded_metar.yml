interactions:
- request:
    body: null
    headers:
      Accept: ['*/*']
      Accept-Encoding: ['gzip, deflate']
      Connection: [keep-alive]
      User-Agent: [python-requests/2.13.0]
    method: GET
    uri: http://avwx.rest/api/metar/KDEN?options=info,translate
  response:
    body: {string: "{\n  \"Altimeter\": \"3009\", \n  \"Cloud-List\": [\n    [\n \
        \     \"FEW\", \n      \"100\"\n    ]\n  ], \n  \"Dewpoint\": \"M04\", \n\
        \  \"Flight-Rules\": \"VFR\", \n  \"Info\": {\n    \"City\": \"Denver\", \n\
        \    \"Country\": \"USA\", \n    \"Elevation\": \"1656\", \n    \"IATA\":\
        \ \"DEN\", \n    \"ICAO\": \"KDEN\", \n    \"Latitude\": \"39.86165639\",\
        \ \n    \"Longitude\": \"-104.6731775\", \n    \"Name\": \"International Airport\"\
        , \n    \"Priority\": \"0\", \n    \"State\": \"CO\"\n  }, \n  \"Meta\": {\n\
        \    \"Cache-Timestamp\": \"Wed, 14 Feb 2018 03:14:11 GMT\", \n    \"Timestamp\"\
        : \"Wed, 14 Feb 2018 03:14:11 GMT\"\n  }, \n  \"Other-List\": [], \n  \"Raw-Report\"\
        : \"KDEN 140253Z 17013KT 10SM FEW100 03/M04 A3009 RMK AO2 SLP191 T00281044\
        \ 51009\", \n  \"Remarks\": \"RMK AO2 SLP191 T00281044 51009\", \n  \"Remarks-Info\"\
        : {\n    \"Dew-Decimal\": \"-4.4\", \n    \"Temp-Decimal\": \"2.8\"\n  },\
        \ \n  \"Runway-Vis-List\": [], \n  \"Station\": \"KDEN\", \n  \"Temperature\"\
        : \"03\", \n  \"Time\": \"140253Z\", \n  \"Translations\": {\n    \"Altimeter\"\
        : \"30.09inHg (1019hPa)\", \n    \"Clouds\": \"Few clouds at 10000ft - Reported\
        \ AGL\", \n    \"Dewpoint\": \"-04C (25F)\", \n    \"Other\": \"\", \n   \
        \ \"Remarks\": {\n      \"51009\": \"3-hour pressure difference: +/- 0.9 mb\
        \ - Increasing, then steady\", \n      \"AO2\": \"Automated with precipitation\
        \ sensor\", \n      \"SLP191\": \"Sea level pressure: 1019.1 hPa\", \n   \
        \   \"T00281044\": \"Temperature 2.8\\u00b0C and dewpoint -4.4\\u00b0C\"\n\
        \    }, \n    \"Temperature\": \"03C (37F)\", \n    \"Visibility\": \"10sm\
        \ (16.1km)\", \n    \"Wind\": \"S-170 at 13kt\"\n  }, \n  \"Units\": {\n \
        \   \"Altimeter\": \"inHg\", \n    \"Altitude\": \"ft\", \n    \"Temperature\"\
        : \"C\", \n    \"Visibility\": \"sm\", \n    \"Wind-Speed\": \"kt\"\n  },\
        \ \n  \"Visibility\": \"10\", \n  \"Wind-Direction\": \"170\", \n  \"Wind-Gust\"\
        : \"\", \n  \"Wind-Speed\": \"13\", \n  \"Wind-Variable-Dir\": []\n}\n"}
    headers:
      Access-Control-Allow-Origin: ['*']
      Content-Length: ['1688']
      Content-Type: [application/json]
      Date: ['Wed, 14 Feb 2018 03:14:10 GMT']
      Server: [Werkzeug/0.14.1 Python/3.6.4]
    status: {code: 200, message: OK}
version: 1
