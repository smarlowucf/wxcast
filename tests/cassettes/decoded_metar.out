At 140253Z the conditions for KDEN are VFR

  Altimeter:  30.09inHg (1019hPa)
     Clouds:  Few clouds at 10000ft - Reported AGL
   Dewpoint:  -04C (25F)
      Other:  
      51009:  3-hour pressure difference: +/- 0.9 mb - Increasing, then
              steady
        AO2:  Automated with precipitation sensor
     SLP191:  Sea level pressure: 1019.1 hPa
  T00281044:  Temperature 2.8°C and dewpoint -4.4°C
Temperature:  03C (37F)
 Visibility:  10sm (16.1km)
       Wind:  S-170 at 13kt

       City:  Denver
    Country:  USA
  Elevation:  5433ft (1656m)
       IATA:  DEN
       ICAO:  KDEN
   Latitude:  39.86165639
  Longitude:  -104.6731775
       Name:  International Airport
   Priority:  0
      State:  CO
